tests/comfy_cli/command/test_distribution.py

```
@@ -584,6 +584,43 @@ def fake_request_json(url, target, *, method="GET", body=None, max_bytes, timeou
     assert ("POST", "https://builder.test/v1/distributions/d1/validate") in calls
 
 
+def test_builder_client_reference_and_update_endpoints(monkeypatch):
+    calls = []
+
+    def fake_request_json(url, target, *, method="GET", body=None, max_bytes, timeout=30.0):
```

Pylint (4.0.6)

[refactor] Too many arguments (6/5)

(R0913)